module/zfs/zfs_vnops_osx.c

```diff
@@ -2691,7 +2691,10 @@ zfs_vnop_pageoutv2(struct vnop_pageout_args *ap)
 	if (error)
 		ubc_upl_abort(upl,  (UPL_ABORT_ERROR | UPL_ABORT_FREE_ON_EMPTY));
 	else
-		ubc_upl_commit_range(upl, 0, a_size, UPL_COMMIT_FREE_ON_EMPTY);
+		ubc_upl_commit_range(upl, 0, a_size,
+		    UPL_COMMIT CLEAR_DIRTY
```

Comment:
There is a missing _ between COMMIT and CLEAR for UPL_COMMIT_CLEAR_DIRTY - doesn't build.

Reply:
Yikes, sorry about that.   You're right.